OpenRA.Game/FileFormats/WavLoader.cs

```diff
@@ -35,23 +35,20 @@ public enum WaveType { Pcm = 0x1, ImaAdpcm = 0x11 }
 
 		bool LoadSound(Stream s)
 		{
-			var type = s.ReadASCII(4);
```

Comment:
Oh, I think I got it quite wrong on how this works. Will drop my changes for now.